modules/end-user-guide/partials/ref_parameters-for-che-editor-yaml.adoc

```diff
@@ -28,23 +28,34 @@ reference: https://__<hostname_and_path_to_a_remote_file>__/che-editor.yaml
 inline:
   schemaVersion: 2.1.0
   metadata:
-    name: theia-ide
-  commands:
-    ...
-  events:
-    ...
+    name: IntelliJ IDEA Community Edition
```

Comment:
see https://github.com/redhat-developer/devspaces/pull/829/files#diff-14148f60b7e8d725c2f196edeb04c6b4750e411561536ecccf55fad177152222R165-R166
and
https://github.com/eclipse-che/che-plugin-registry/pull/1441/files#diff-4526f5cc968bddb01b7aaa7b95bbf3be602a7cfe9b7a69230ecbef43751732b4R640

Reply:
Two options here:
`IntelliJ IDEA Community`
or
`JetBrains IntelliJ IDEA Community IDE`
Which one do you prefer?

Reply:
If you use `JetBrains IntelliJ IDEA Community IDE` you'll make the brand/trademark people happier. 

And by not using the `for {prod}' suffix (upstream) or `{prod} with` prefix (downstream), transformation of doc from up to down will be simpler.

Reply:
Thank you, Nick!